server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/EmptyDialect.java

```diff
@@ -12,22 +12,24 @@
  *    Broadcom, Inc. - initial API and implementation
  *
  */
-package org.eclipse.lsp.cobol.core.engine.flavors;
+package org.eclipse.lsp.cobol.core.engine.dialects;
 
 import com.google.common.collect.ImmutableList;
+import org.eclipse.lsp.cobol.core.messages.MessageService;
 import org.eclipse.lsp.cobol.core.model.ResultWithErrors;
 
-/** Dummy flavor which is doing nothing */
-public class EmptyFlavor {
+/** Dummy dialect which is doing nothing */
+public class EmptyDialect implements CobolDialect {
 
   /**
    * Return the same text from the input and empty lists of nodes and errors.
    *
    * @param uri document URI
    * @param text document text
-   * @return the flavor processing result
+   * @param messageService error message service
+   * @return the dialect processing result
    */
-  public static ResultWithErrors<FlavorOutcome> processText(String uri, String text) {
-    return new ResultWithErrors<>(new FlavorOutcome(text, ImmutableList.of()), ImmutableList.of());
+  public ResultWithErrors<DialectOutcome> processText(String uri, String text, MessageService messageService) {
+    return new ResultWithErrors<>(new DialectOutcome(text, ImmutableList.of()), ImmutableList.of());
```

Comment:
You can make it a default method in the CobolDialect interface

Reply:
Indeed!